cartographer_ros/cartographer_ros/map_builder_bridge.h

```diff
@@ -44,32 +44,33 @@ class MapBuilderBridge {
     TrajectoryOptions trajectory_options;
   };
 
-  MapBuilderBridge(const NodeOptions& node_options, tf2_ros::Buffer* tf_buffer);
+  MapBuilderBridge(const NodeOptions &node_options, tf2_ros::Buffer *tf_buffer);
```

Comment:
I have, on the whole cartographer_ros codebase (and I have re-run it just now to confirm), and it outputs this. Should I revert these changes nevertheless?